### DISCUSSION TITLE
In explore, showing "published discussion" instead of "starred"

### DISCUSSION THREAD

**Discussion by Chromosore** [community/community#24061] [Apps, API and Webhooks] [answered]
<p>In the explore view some cards are shown with the text “[user] published a discussion” even if the icon shows the profile picture with a tiny star</p>
<p>I’ve checked on the website and card there indeed shows “[user] starred [repo]”.</p>
<p>I use GitHub on Android version 1.73.0<br>
I can provide screenshots of the app and the website showing the said cards if needed.</p>

**Comment by jfullstackdev** [comment]
<p>you mean they are not displaying the same thing  ?</p>

**Comment by Chromosore** [comment, marked as answer]
<p>Oh wait this issue was already covered in the new GitHub discussions.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/orgs/github-community/discussions/19792">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/orgs/github-community/discussions/19792" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/37866106/181081770-0790cbb6-8827-4e7a-96fd-df83ac2e35a0.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/d/dde685dc0d767cbb2193e8cd6bea9eff38ef039b_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/d/dde685dc0d767cbb2193e8cd6bea9eff38ef039b_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/d/dde685dc0d767cbb2193e8cd6bea9eff38ef039b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/d/dde685dc0d767cbb2193e8cd6bea9eff38ef039b_2_10x10.png"></div>

<h3><a href="https://github.com/orgs/github-community/discussions/19792" target="_blank" rel="noopener nofollow ugc">In the "explore" tab on mobile when someone stars a repository it says...</a></h3>

  <p>When you tap the explore tab, you see "Started a discussion" instead of "Starred a repository".</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
